js/views/AddressBarIndicators.js

```diff
@@ -70,6 +79,10 @@ export default class extends baseVw {
   }
 
   render() {
+    super.render();
```

Comment:
Nope. Removed.

Reply:
It's not needed if your not doing any getCachedEl calls, but I generally throw it on most views (unless I forgot) because then it avoids tricky bugs where you do put in a getCachedEl, but forget the super.render().